tests/test_advanced_filters.py

```
@@ -0,0 +1,2458 @@
+    response = client.post(
+        f"/v2/workspaces/{workspace_name}/peers/{peer_name}/messages/list",
+        json={"filter": {"metadata": {"calculation": "0.1 + 0.2"}}},
+    )
+    assert response.status_code == 200
+    data = response.json()
+    contents = [item["content"] for item in data["items"]]
+    assert "Point one plus point two" in contents
+
+
+@pytest.mark.asyncio
+async def test_mixed_type_comparisons(client: TestClient):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_mixed_type_comparisons`

Add return type annotation: `None`

(ANN201)